### PR TITLE
Update instructions about how to build docs

### PR DESCRIPTION
**Pull request by bmw**
The resulting documentation looks like 
Screen Shot 2019-11-26 at 12 16 54 PM where "local copy" links to https://certbot.eff.org/docs/contributing.html#running-a-local-copy-of-the-client.

### CODE REVIEW

File: certbot/docs/contributing.rst [hunk 1]
```diff
@@ -520,19 +520,21 @@ during the next release.
 Updating the documentation
 ==========================
 
-In order to generate the Sphinx documentation, run the following
-commands:
+Many of the packages in the Certbot repository have documentation in a
```
Comment (adferrand) [inline certbot/docs/contributing.rst]: Trying to remove the number of `in` with my "English" : Most of the packages in the Certbot repository have a documentation stored as a ``docs`` directory. This directory is located under the root path of the package.

File: certbot/docs/contributing.rst [hunk 2]
```diff
@@ -520,19 +520,21 @@ during the next release.
...
+``docs/`` directory in the top level directory for the package. For instance,
+Certbot's documentation is under ``certbot/docs``.
 
-.. code-block:: shell
-
-   make -C docs clean html man
+To build this any of this documentation, make sure you have followed the
```
Comment (adferrand) [inline certbot/docs/contributing.rst]: To rebuilt the documentation of a package...

File: certbot/docs/contributing.rst [hunk 3]
```diff
@@ -520,19 +520,21 @@ during the next release.
...
+instructions to set up a `local copy`_ of Certbot including activating the
+virtual environment and then ``cd`` to the docs directory you want to build and
```
Comment (adferrand) [inline certbot/docs/contributing.rst]: , then ``cd``...

**Comment by bmw** [comment]
I didn't take any of your suggestions as is, but let me know if the new wording addresses your concerns.

**Comment by bmw** [comment]
@adferrand, since you already looked at this once, are you available and interested in giving this another review?

**Pull request merged**